Makefile

```diff
@@ -10,7 +10,6 @@ all: $(DEPS)
 
 $(DCURL_LIB): $(DCURL_DIR)
 	git submodule update --init $^
-	git submodule update --remote $^
```

Comment:
Why did you change this line?

Reply:
@wusyong said we should pin to v0.1.0 at this moment, so I removed this line for fixing dependency on v0.1.0

Reply:
Ask @wusyong for finding proper solutions for tracking certain versions.

Reply:
ok!

Reply:
Since dcurl enabled semantic versioning, it should stay on certain version unless update required. That means we shall not update to latest commit from remote. I think we should remove this within or without this PR.